controllers/openstackcluster_controller.go

```diff
@@ -740,6 +740,33 @@ func reconcilePreExistingNetworkComponents(scope *scope.WithLogger, networkingSe
 		setClusterNetwork(openStackCluster, network)
 	}
 
+	// add pre-existing router to status if router spec is defined
```

Comment:
I'm not sure about this comment. Either make it better or remove it, thanks.

Reply:
thanks for the comment, I removed it because other code snmippets like that are not commented either.